portfolio/src/main/webapp/me.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <h1>My Portfolio</h1>
+    <header>
+    <div class="topnav">
+      <a class="active" href="index.html">Home</a>
+      <a href="me.html">About Me</a>
+      <a href="travels.html">Travels</a>
+    </div>
+    <div>
+      <!-- TODO: About Me page's content -->
```

Comment:
Mark your `TODO` with your ldap or username: https://google.github.io/styleguide/htmlcssguide.html#Action_Items

Also (opt), please make your TODOs imperative instead of declarative.

`TODO(jdstriker731): Add content for 'About Me' page.`